src/Beta/Model/AccessAction.php

```diff
@@ -12,7 +12,7 @@
 * @version   GIT: 1.4.0
 * @link      https://graph.microsoft.io/
 */
-namespace Microsoft\Graph\Model;
+namespace Microsoft\Graph\Beta\Model;
```

Comment:
Namespace change. What should be the proper namespace for beta classes?

Reply:
Do we know how this happened?

Reply:
It is supposed to have beta in the namespace. This must have been run without beta at some point. I don't know exactly how this happened.